### ISSUE TITLE
The `\d` command is a bit broken

### ISSUE THREAD

**Issue by vpetrovykh** [edgedb/edgedb#1218]
Currently you can do this:
```
db> \d all
function std::all(vals: SET OF std::bool) ->  std::bool {
    volatility := 'IMMUTABLE';
    using sql
;};


db> CREATE FUNCTION all() -> bool USING EdgeQL 'SELECT True';
CREATE
db> \d all
function default::all() ->  std::bool using (SELECT True);


db> CREATE TYPE all;
CREATE
db> \d all
function default::all() ->  std::bool using (SELECT True);


db> \d default::all
function default::all() ->  std::bool using (SELECT True);
```

The function `default::all` successfully masks the type `default::all` (same happens if you had an abstract link `default::all`). I think that other cases of masking are impossible as we require names for types, links, props, and constraints to be universally unique, but only functions are exempt from that. We need some way to handle this, maybe some optional flags for the `\d` command that make it target specific schema types.

**Comment by 1st1** [comment]
Yeah, we should mirror `\l*` commands to `\d*`, e.g. `\df`, `\dt`, etc.

When there's a name collision for `\d` we should probably list all descriptions.

**Comment by 1st1** [comment]
@vpetrovykh Please verify if there's anything left to do here.

**Comment by vpetrovykh** [comment]
This is definitely still an issue as of `EdgeDB 1.0-alpha.3+dev.214.gd0f51837`.

**Comment by elprans** [comment]
I believe this has been fixed in #1432
